promql/parser/generated_parser.y

```diff
@@ -380,6 +382,18 @@ offset_expr: expr OFFSET duration
                 | expr OFFSET error
                         { yylex.(*parser).unexpected("offset", "duration"); $$ = $1 }
                 ;
+/*
+ * @ modifiers.
+ */
+
+instant_expr: expr AT number
```

Comment:
Instant expression is confusing here as a name, as we've already the notions of instant vectors and instant queries - of which this is neither.